pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/visualization/PiTestReports.kt

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023
+
+package com.amos.pitmutationmate.pitmutationmate.visualization
+
+import com.amos.pitmutationmate.pitmutationmate.reporting.XMLParser
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.components.JBScrollPane
+import com.intellij.ui.table.JBTable
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+import java.awt.BorderLayout
+import java.awt.Component
+import java.awt.Dimension
+import javax.swing.JComponent
+import javax.swing.JPanel
+import javax.swing.JTable
+import javax.swing.ListSelectionModel
+import javax.swing.table.DefaultTableCellRenderer
+import javax.swing.table.DefaultTableModel
+
+private class cellRenderer : DefaultTableCellRenderer() {
+
+    override fun getTableCellRendererComponent(
+        table: JTable?,
+        value: Any?,
+        isSelected: Boolean,
+        hasFocus: Boolean,
+        row: Int,
+        column: Int
+    ): Component {
+        if (value is PiTestClassReport) {
+            return value
+        } else if (value is CustomProgressBar) {
+            return value
+        } else if (value is JBLabel) {
+            return value
+        } else {
+            return super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column)
+        }
+    }
+}
+
+fun createTable(data: Array<Array<out JComponent>>, columnNames: Array<String>): JBTable {
+    val model = object : DefaultTableModel(data, columnNames) {
+        override fun isCellEditable(row: Int, column: Int): Boolean {
+            return false
+        }
+    }
+
+    val table = JBTable(model)
+
+//    table.setRowHeight(lineCoverageBar.height + 2)
+    table.tableHeader.reorderingAllowed = false
+    table.tableHeader.resizingAllowed = false
+    table.tableHeader.font = UIUtil.getLabelFont()
+
+    table.border = JBUI.Borders.empty()
+    table.setShowGrid(false)
+    table.setSelectionMode(ListSelectionModel.SINGLE_SELECTION)
+    table.columnSelectionAllowed = false
+    table.intercellSpacing = Dimension(0, 0)
+
+    val firstColumnWidth = table.tableHeader.getFontMetrics(table.tableHeader.font).stringWidth(" Pit Test Coverage Report ") + 5
```

Comment:
Could you write +10 instead of +5. Otherwise the title "Pit Test Coverage Report" is not fully shown.